.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.6'
+          - 'lts'
```

Comment:
```suggestion
          - 'min'
          - 'lts'
```
(They are not identical)

Reply:
What's min?

Reply:
It tests the minimum compatible version.